src/NServiceBus.Core/Routing/DistributionPolicy.cs

```diff
@@ -3,14 +3,22 @@ namespace NServiceBus
     using System.Collections.Concurrent;
     using Routing;
 
-    class DistributionPolicy : IDistributionPolicy
+    /// <summary>
+    /// Allows to configure distribution strategies.
+    /// </summary>
+    public class DistributionPolicy : IDistributionPolicy
```

Comment:
should we rename it to `DefaultDistributionStrategy` instead to avoid confusion about retrieving the `DistributionPolicy` in the feature config to set a custom strategy vs. setting a new policy?